apps/insights/src/components/PriceFeeds/index.tsx

```diff
@@ -124,6 +124,35 @@ export const PriceFeeds = async () => {
           />
         </UnstyledTabPanel>
       </UnstyledTabs>
+      <Card
+        className={styles.trademarkDisclaimerCard}
+        title="Trademark Disclaimer"
+      >
```

Comment:
This does not look good as a card, so let's remove the `Card` component wrapping this.

Instead, please add a header that says "Trademark Disclaimer" inside the div with the `trademarkDisclaimerContent` class.  Give the header text a bit larger size, and some space between it and the paragraph.